snapcraft/plugins/catkin.py

```diff
@@ -889,9 +889,11 @@ def find(self, package_name):
 
     def _run(self, arguments):
         with tempfile.NamedTemporaryFile(mode='w+') as f:
-            lines = ['export PYTHONPATH={}'.format(os.path.join(
-                self._catkin_install_path, 'usr', 'lib', 'python2.7',
-                'dist-packages'))]
+            lines = ['export PYTHONPATH={}:{}'.format(
+                os.path.join(self._catkin_install_path, 'usr', 'lib',
+                             'python2.7', 'dist-packages'),
+                os.path.join(self._catkin_install_path, 'opt', 'ros',
+                             'kinetic', 'lib', 'python2.7', 'dist-packages'))]
```

Comment:
Ok, so you are correct that when catkin is part of the underlay, _Catkin.find works as intended. However, I am including the underlay in the snap (to use custom, pre-built tarballs of some ROS packages), and when catkin is part of the underlay, it creates a staging conflict because the catkin deb (a different version) is automatically installed as part of the catkin plugin.

What should be the correct way to build a single snap with an underlay?